config/jobs/kubernetes-sigs/sig-windows/release-1.18-windows.yaml

```diff
@@ -164,13 +164,13 @@ periodics:
       - --aksengine-win-binaries
       - --aksengine-deploy-custom-k8s
       # Specific test args
-      - --test_args=--node-os-distro=windows --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Slow\]|Guestbook.application.should.create.and.stop.a.working.application
+      - --test_args=--node-os-distro=windows --ginkgo.focus=$(GINKGO_FOCUS_PARALLEL) --ginkgo.skip=$(GINKGO_SKIP_PARALLEL)
       - --ginkgo-parallel=4
       securityContext:
         privileged: true
   annotations:
-    testgrid-dashboards: sig-windows-releases, sig-windows-azure, sig-release-1.18-informing
-    testgrid-tab-name: aks-engine-azure-1-18-windows
+    testgrid-dashboards: sig-release-1.18-informing, sig-windows-1.18-release
+    testgrid-tab-name: aks-engine-windows-dockershim
```

Comment:
These should probably still have the version number (1.18) in the tab name?

Reply:
The reason why I didn't include the version name is that it's already encoded in the dashboard name

Reply:
looking at a sig-release dashboard I see most have the release name encoded: https://testgrid.k8s.io/sig-release-1.20-informing.  It's minor but would be helpful if it does get duplicated across dashboards

Reply:
That makes sense. Let me append the version to the end of the tab name